main_f4.c

```diff
@@ -127,12 +134,6 @@ board_test_force_pin()
 	volatile unsigned samples = 0;
 	volatile unsigned vote = 0;
 
-	/* (re)configure the force BL pins */
-	rcc_peripheral_enable_clock(&BOARD_FORCE_BL_CLOCK_REGISTER, BOARD_FORCE_BL_CLOCK_BIT);
-	gpio_mode_setup(BOARD_FORCE_BL_PORT, GPIO_MODE_INPUT, BOARD_FORCE_BL_PULL, BOARD_FORCE_BL_PIN_IN);
-	gpio_mode_setup(BOARD_FORCE_BL_PORT, GPIO_MODE_OUTPUT, GPIO_PUPD_NONE, BOARD_FORCE_BL_PIN_OUT);
-	gpio_set_output_options(BOARD_FORCE_BL_PORT, GPIO_OTYPE_PP, GPIO_OSPEED_100MHZ, BOARD_FORCE_BL_PIN_OUT);
-
```

Comment:
I moved these initialisations into the board_init function to be consistent with where all of the other peripherals are set.